test/OpenTelemetry.Instrumentation.AspNetCore.Tests/IncomingRequestsCollectionsIsAccordingToTheSpecTests.cs

```diff
@@ -39,114 +39,106 @@ public IncomingRequestsCollectionsIsAccordingToTheSpecTests(WebApplicationFactor
     }
 
     [Theory]
-    [InlineData("/api/values", null, "user-agent", 503, "503")]
-    [InlineData("/api/values", "?query=1", null, 503, null)]
+    [InlineData("/api/values", null, "user-agent", 200, null)]
```

Comment:
Why update the test cases? Do you simply want to add test cases for `200`?

Reply:
I did not update it in this PR. It was already there, it showing up due to file rename

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/4986